### PR TITLE
Make x-ray background transparent

### PR DESCRIPTION
**Pull request by linouk23**
### Desc
As a UI developer, I would like the background color of x-rays tiles to be transparent, so that I can easily overlay them on other map data.

### CODE REVIEW

**Comment by pdubroy** [comment]
If you only turn white pixels to transparent, I'm pretty sure we will see aliasing artifacts around the edges. Is it possible to instead make all pixels transparent? E.g., starting with setting `alpha` to 0 in `get_pixel_color`.

File: xray/src/generation.rs [hunk 1]
```diff
@@ -315,11 +315,16 @@ pub fn build_parent(children: &[Option<image::RgbImage>]) -> image::RgbImage {
...
     let child_size_px = child_size_px.expect("No children passed to 'build_parent'.");
-    let mut large_image = image::RgbImage::from_pixel(
+    let mut large_image = image::RgbaImage::from_pixel(
         child_size_px * 2,
         child_size_px * 2,
-        image::Rgb {
-            data: [255, 255, 255],
+        image::Rgba {
+            data: [
+                TRANSPARENT.to_u8().red,
```
Comment (pdubroy) [inline xray/src/generation.rs]: Can you make it a comment line option whether to make the background white or transparent?

**Comment by pdubroy** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user pdubroy. Merge queue now has a length of 1.

**Pull request merged**